docs/d3-geo/projection.md

```diff
@@ -101,21 +101,21 @@ projection.fitSize([width, height], object);
 
 [Source](https://github.com/d3/d3-geo/blob/main/src/projection/index.js) · A convenience method for [*projection*.fitSize](#projection_fitSize) where the width is automatically chosen from the aspect ratio of *object* and the given constraint on *height*.
 
-## Raw projections
+## Raw projections {#raw-projections}
```

Comment:
This one shouldn’t be necessary since it should be the default anchor for this heading.